### PR TITLE
Add representative

### PR DESCRIPTION
**Pull request by TimBrowne066**
In this commit, we added a feature branch with a complete MVC rails app with Representatives, Reviews. and Comments.  We want to merge with the User Authentication branch to finish the app.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,11 +10,45 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
+ActiveRecord::Schema.define(version: 20170905183319) do
+=======
```
Comment (DRBragg) [inline db/schema.rb]: this is where codeship is failing... hit it with a hammer?

**Comment by DRBragg** [comment]
fixed it by dropping DB and re-setting it up

File: app/controllers/representatives_controller.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    @representative = Representative.create(representative_params)
+    if @representative.save
+      redirect_to @representative, notice: "Representative added successfully"
+    else
+      flash[:alert] = @representative.errors.full_messages.join(', ')
+      render :new
+    end
+  end
+
+  def edit
```
Comment (smkopp92) [inline app/controllers/representatives_controller.rb:29]: Generally you do not want to leave empty actions in your controller like this. Leave them out until they are implemented

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -3,12 +3,16 @@
   <head>
     <title>On Blast</title>
     <%= csrf_meta_tags %>
-
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
   </head>
-
   <body>
+<<<<<<< HEAD
```
Comment (smkopp92) [inline app/views/layouts/application.html.erb:10]: You have some leftover junk from your merge conflicts, please remove the git syntax

**Pull request merged**